src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -67,113 +69,153 @@ public static ParseUDFsResult ParseUDFsScript(string script, CultureInfo loc = n
             return parser.ParseUDFs(script);
         }
 
+        /// <summary>
+        /// If the token is of the right type, it returns it and moves forward.
+        /// This does not post an error if the tok is of the wrong type.
+        /// </summary>
+        /// <param name="tid"></param>
+        /// <returns></returns>
+        private Token TokPlainEat(TokKind tid)
```

Comment:
Do we need this function? There are existing ways of checking to see if the next token is of a specific type.

Reply:
Removed.